PF-Assignment1/2.RemoveDuplicate/Assignment2Test.java

```diff
@@ -0,0 +1,45 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class Assignment2Test {
+	Assignment2 obj = new Assignment2();
+	
+	@Test
+	  public void removeDuplicateTest1() {
```

Comment:
Following naming conventions as suggested. Like name for test method should be 'testRemoveDuplicateShouldRemoveDuplicates'